Breeder/BR_Util.cpp

```diff
@@ -1078,6 +1078,22 @@ bool TrimItem_UseNativeTrimActions(MediaItem* item, double start, double end, bo
 	if (!item)
 		return false;
 
+	// Fetch the current start and length for the item
+	double itemPos = GetMediaItemInfo_Value(item, "D_POSITION");
+	double itemLen = GetMediaItemInfo_Value(item, "D_LENGTH");
+
+	// If start is negative then keep the current start position
+	if(start < 0)
+	{
+		start = itemPos;
+	}
+
+	// If end is negative then keep the current end position
+	if(end < 0)
+	{
+		end = itemPos + itemLen;
+	}
```

Comment:
Nice feature!

There was already a check for `if (start < 0)` a few lines below, which is now redundant (should be removed).

A small improvement suggestion to match the style of the surrounding code:
```suggestion
	if (start < 0)
		start = itemPos;

	// If end is negative then keep the current end position
	if (end < 0)
		end = itemPos + itemLen;
```

Reply:
Good catch!  Thanks for the review, I've pushed your requested changes.